modules/ROOT/pages/debugging-kernel-crashes.adoc

```diff
@@ -1,10 +1,10 @@
 = Debugging kernel crashes using kdump
 
-. Memory must be reserved for the crash kernel during booting of the first kernel. Kernel arguments can be provided like this:
+. Memory must be reserved for the crash kernel during booting of the first kernel. `crashkernel=auto` is likely insufficient memory on Fedora CoreOS. It is recommended to start testing with 300M. Kernel arguments can be provided like this:
```

Comment:
(Aside: I wonder what heuristics `auto` uses and whether we need to tweak it so it also Just Works on FCOS.)

Reply:
I think the heuristics can be found here. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/installing-and-configuring-kdump_managing-monitoring-and-updating-the-kernel#memory-requirements-for-kdump_supported-kdump-configurations-and-targets
Not sure if kdump just assumes you're using `dracut-squash`, because FCOS seems to require way more reserved memory than it should.

Reply:
Ahh gotcha. Yeah that's likely the case.

We can consider adding dracut-squash if there's strong requests for it, though it seems like the memory requirement is just slightly higher, which seems like a good compromise for keeping those packages out of the host for a feature that most people likely won't turn on.